astropy/modeling/core.py

```diff
@@ -1453,6 +1452,23 @@ def _format_str(self, keywords=[]):
 
         return '\n'.join(parts)
 
+    def _initialize_bounding_box_default(self):
```

Comment:
I think @embray (https://github.com/astropy/astropy/pull/3909#discussion_r37435920) suggested a different solution. You can define `bounding_box_default` in the `Model` class, but it should just raise an `NotImplementedError`. Specific models, that inherit from `Model`then override this method.